### DISCUSSION TITLE
The user=tenent_001 is Permission denied

### DISCUSSION THREAD

**Discussion by xgxizz** [apache/dolphinscheduler#11807] [Q&A] [answered]
I created a spark flowtask, but the user=tennet_001 is Permission denied.
My  HADOOP_HOME is chown to xgx.
my problem is in the picture bellow:
![image](https://user-images.githubusercontent.com/46493228/188604846-4a7314e7-d7bb-4d37-ab02-81c7ff801b8b.png)
How can I solve this problem?

**Comment by DarkAssassinator** [comment]
because the use tenent_001 do not have the write perssion.  u can check `drwrr-xr-x`

**Reply by xgxizz** [reply]
That is to say, after creating a tenant, must I manually create a user with the same name on the operating system? And this user has write permission to HDFS?

**Comment by xgxizz** [comment, marked as answer]
I have solved this problem.
By default, HDFS enables permission verification. Add the following configuration to the Hadoop configuration file hdfs-site.xml.
` <property>
 　　           <name>dfs.permissions.enabled</name>
 　　           <value>false</value>                          
        </property>`
